doc/DeveloperGuide.md

```diff
@@ -94,6 +94,10 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. The system should be usable to someone familiar with typing commands.
+6. The product is not required to have extravagant GUI effects.
```

Comment:
This is not a requirement. A similar NFR would be something like: `Should prioritise speed over extravagant GUI effects`.